doc/tracepath.xml

```diff
@@ -162,28 +162,28 @@ root@mops:~ # tracepath -6 3ffe:2400:0:109::2
  2:  3ffe:2400:0:109::2               463.514ms reached
      Resume: pmtu 1480 hops 2 back 2
     </literallayout>
-    <para>The first column shows TTL of the probe, followed by
-    colon. Usually value of TTL is obtained from reply from
-    network, but sometimes reply does not contain necessary
+    <para>The first column shows the TTL of the probe, followed by
+    colon. Usually the value of TTL is obtained from the reply from
+    the network. But sometimes it does not contain the necessary
     information and we have to guess it. In this case the number is
     followed by ?.</para>
     <para>The second column shows the network hop, which replied to
-    the probe. It is either address of router or word [LOCALHOST],
-    if the probe was not sent to the network.</para>
+    the probe. It is either the address of the router or the word
+    [LOCALHOST], if the probe was not sent to the network.</para>
     <para>The rest of line shows miscellaneous information about
-    path to the corresponding network hop. As rule it contains
-    value of RTT. Additionally, it can show Path MTU, when it
+    the path to the corresponding network hop. At least it contains
+    the value of RTT. Additionally, it can show Path MTU, when it
     changes. If the path is asymmetric or the probe finishes before
-    it reach prescribed hop, difference between number of hops in
-    forward and backward direction is shown following keyword
-    async. This information is not reliable. F.e. the third line
-    shows asymmetry of 1, it is because the first probe with TTL of
-    2 was rejected at the first hop due to Path MTU
+    it reaches the prescribed hop, the difference between number of
+    hops in forward and backward direction is shown following the
+    keyword "async". This information is not reliable. e.g. the
```

Comment:
`This information is not reliable, e.g.` (comma instead of dot).

Reply:
`the difference between number of hops in forward and backward direction is shown following the keyword "async". `
Not sure if it should be `shown with following the keyword "async".` or `shown by the keyword "async".`, but maybe I don't understand the meaning.

Reply:
According to my understanding it means `there is a number following the word "async"`.